src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -30,8 +27,12 @@ public unsafe partial class RaspberryPi3Driver : GpioDriver
         private const string DeviceTreeRanges = "/proc/device-tree/soc/ranges";
         private const string ModelFilePath = "/proc/device-tree/model";
 
-        private UnixDriver _sysFSDriver = null;
         private readonly IDictionary<int, PinMode> _sysFSModes = new Dictionary<int, PinMode>();
```

Comment:
Just wondering, is it a rule that constants need to be on their own and that they have to be first?

Reply:
I think it's this order (second rule is whatever is most public goes first):
- constants
- fields
- constructors
- methods
- nested types

Reply:
Yes, that's the rule that's enabled now, except that it is 
- constants
- readonly fields
- read-write fields
- constructors
- properties
- nested types (classes)
There are other similar rules in the "ordering" category https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/OrderingRules.md, but I think this one is the most useful rule (and there where actually few violations of it). 

The other rule, SA1202, ElementsMustBeOrderedByAccess is disabled for now (and I personally think it is not helping anything, and requires moving stuff around in files if the visibility changes)

Reply:
I thought the properties go before constructors, I got impression everyone does that at least 😄

Reply:
SA1202 let's consider afterwards, more important to get bulk ASAP to not affect pending PRs too much